### PR TITLE
feat: banner image change by language

### PR DESCRIPTION
**Pull request by GusBaamonde**
Banner image change (notebook) by language
related to https://github.com/FromDoppler/Doppler-UI-System/pull/104 and
http://jira.makingsense.com/browse/DBR-283

- Signup (ES/EN)
- Login (ES/EN)
- Forgot password (ES/EN)

change-images

### CODE REVIEW

File: src/i18n/en.js [hunk 1]
```diff
@@ -1,3 +1,7 @@
+import signupBanner from '../img/signup-en.png';
+import loginBanner from '../img/login-en.png';
+import forgotPasswordBanner from '../img/login-en.png';
```
Comment (andresmoschini) [inline src/i18n/en.js]: What do you think about adding _ImagePath_ to the name of these variables?

```suggestion
import forgotPasswordBannerImagePath from '../img/login-en.png';
```

File: src/i18n/es.js [hunk 2]
```diff
@@ -210,6 +217,7 @@ export default {
     sign_up: `Email, Automation & Data Marketing`,
     sign_up_sub: `Atrae, Convierte y Fideliza. Envíos ilimitados y gratis hasta 500 Suscriptores.`,
     thanks_for_registering: `Gracias por registrarte`,
+
```
Comment (andresmoschini) [inline src/i18n/es.js]: Could you remove this empty line?

File: src/i18n/en.js (hunk 1 above)
Comment (andresmoschini) [inline src/i18n/en.js]: If the file is the same, please use only one variable:

```js
import signupBanner from '../img/signup-en.png';
import loginBanner from '../img/login-en.png';

// . . .

forgot_password: {
  image_path: `${loginBanner}`,

// . . . 
```

**Comment by andresmoschini** [comment]
👍

File: src/i18n/en.js (hunk 1 above)
Comment (GusBaamonde) [reply src/i18n/en.js]: ok

File: public/index.html [hunk 3]
```diff
@@ -31,7 +31,7 @@
     <div id="root">
       <div class="wrapper-loading">
         <div class="loading-page">
-          <p class="flash">&nbsp;</p>
+          <p class="flash">Loading...</p>
```
Comment (fcoronelmakingsense) [inline public/index.html]: its ok only in english?

**Pull request merged**